src/backend/src/lib.rs

```diff
@@ -436,6 +439,22 @@ fn list_custom_tokens() -> Vec<CustomToken> {
     read_state(|s| s.custom_token.get(&stored_principal).unwrap_or_default().0)
 }
 
+async fn bitcoin_ecdsa_public_key_of(canister_id: Principal, network: BitcoinNetwork) -> Vec<u8> {
+    let name = read_config(|s| s.ecdsa_key_name.clone());
```

Comment:
We can use the same name for both ETH and Bitcoin?

Reply:
same comment as above